libdocker/libdocker.go

```diff
@@ -185,9 +186,8 @@ func AllInspectedContainers(endpoint string) ([]*CompleteDockerContainer, error)
 		go func(container *CompleteDockerContainer) {
 			defer wg.Done()
 
-			fullDescContainer, err := client.InspectContainer(shortDescContainer.ID)
+			fullDescContainer, err := client.InspectContainer(container.ID)
```

Comment:
shortDescContainer.ID comes from client.ListContainer(), fullDescContainer.ID comes from client.InspectContainer(), both represent the ID of the container being listed / inspected. I think this is ok